cmd/registry/cmd/get/get.go

```diff
@@ -71,8 +71,8 @@ func Command() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVar(&filter, "filter", "", "Filter selected resources")
-	cmd.Flags().StringVarP(&output, "output", "o", "name", "Output type (name, yaml, contents)")
+	cmd.Flags().StringVar(&filter, "filter", "", "filter selected resources")
+	cmd.Flags().StringVarP(&output, "output", "o", "name", "output type (name, yaml, contents)")
```

Comment:
Everywhere else we use an enum style of (name|yaml|contents). We should pick one.

Reply:
Thanks, I'm adding a check that reports this:
```
    root_test.go:85: "registry get" "output" flag usage seems to contain an enum "(name, yaml, contents)", which should be of the form (a|b|c).
```

with a fix to `registry get`.